src/components/Layout.tsx

```diff
@@ -96,15 +99,6 @@ const Layout: React.FC<IProps> = ({
   // Exit Zen Mode on 'esc' click
   useKeyPress(`Escape`, () => handleZenModeChange(false))
 
-  // set isDarkTheme based on browser/app user preferences
-  useEffect(() => {
-    if (localStorage && localStorage.getItem("dark-theme") !== null) {
-      setIsDarkTheme(localStorage.getItem("dark-theme") === "true")
-    } else {
-      setIsDarkTheme(window.matchMedia("(prefers-color-scheme: dark)").matches)
-    }
-  }, [])
-
```

Comment:
@pettinarip So am I correct the users browser preference will still be accounted for here?

Reply:
Yes, we are using this flag `useSystemColorMode` for that. More on this here: https://chakra-ui.com/docs/styled-system/color-mode#updating-the-theme-config